cicecore/cicedyn/analysis/ice_history.F90

```diff
@@ -2778,8 +2774,7 @@ subroutine accum_hist (dt)
            worka(:,:) = c0
            do j = jlo, jhi
            do i = ilo, ihi
-              if (aice_init(i,j,iblk) > puny) &
-                 worka(i,j) = aice(i,j,iblk)*(Tbot(i,j,iblk)/aice_init(i,j,iblk)+Tffresh)
+               worka(i,j) = aice(i,j,iblk)*Tbot(i,j,iblk)
```

Comment:
could shift this line 1 character left

Reply:
This is just metadata in the netcdf files. We don't have this for binary files?